### DISCUSSION TITLE
Allow emoji reactions when commenting is locked!

### DISCUSSION THREAD

**Discussion by babakfp** [community/community#84925] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi
Sometimes a conversation may be locked for various reasons, when this happens, emoji reactions also get locked, so it's impossible to react with emojis. This is very frustrating because it prevents people from expressing themselves. It's safe to allow emoji reactions even if the conversation is locked. I don't see any negative result in allowing people to react with emojis.
Please **upvote** this so the GitHub team can see and prioritize it.
Thanks

**Comment by lonnywong** [comment]
I want to create a locked issue that disables comments, disables references, and only allows emoji votes. It would be great if there was an option to do this when locking an issue.
